go.mod

```diff
@@ -185,10 +185,10 @@ require (
 	go.mongodb.org/mongo-driver v1.13.1 // indirect
 	go.opencensus.io v0.24.0 // indirect
 	go.opentelemetry.io/otel/metric v1.22.0 // indirect
-	golang.org/x/crypto v0.31.0 // indirect
+	golang.org/x/crypto v0.32.0 // indirect
```

Comment:
I think 0.31.0 for x/crypto is ok. Did this change based on bumping x/net?

Reply:
Yes, also I wanted latest of this just as a good practice.